wled00/FX_fcn.cpp

```diff
@@ -1404,18 +1418,19 @@ void WS2812FX::show() {
   // avoid race condition, capture _callback value
   show_callback callback = _callback;
   if (callback) callback();
+  unsigned long showNow = millis();
 
   // some buses send asynchronously and this method will return before
   // all of the data has been sent.
   // See https://github.com/Makuna/NeoPixelBus/wiki/ESP32-NeoMethods#neoesp32rmt-methods
   BusManager::show();
 
-  unsigned long showNow = millis();
   size_t diff = showNow - _lastShow;
   size_t fpsCurr = 200;
   if (diff > 0) fpsCurr = 1000 / diff;
   _cumulativeFps = (3 * _cumulativeFps + fpsCurr +2) >> 2;   // "+2" for proper rounding (2/4 = 0.5)
   _lastShow = showNow;
+  _lastServiceShow = showNow;
```

Comment:
Unnecessary as it is overwritten in `service()`.

Reply:
Not always. With network "override" strip.show() can get called directly, calling "service()" much later due to strip.trigger(). So this was a backup solution when strip.service is not involved.

Reply:
Having network data override built in effect calculation should have no effect on FPS. As in such case FPS is determined by the caller.

If on the other hand you want to get actual FPS LEDs are updating, then only use one in `show()`.

Reply:
> If on the other hand you want to get actual FPS LEDs are updating, then only use one in `show()`.

That's exactly what I didn't want 😉 
I'll remove this assignment so that only service() will update `_lastServiceShow`.

Reply:
removed.